host/src/Makefile

```diff
@@ -0,0 +1,19 @@
+ifneq ($(KERNELRELEASE),)
```

Comment:
This is actually a comment for `host/src/Kconfig`.  Let's remove the "xap", "xaptum", and "router card" references.  Can the config option  be called  `HSS` instead?  (Assuming the device side option is `HSS_GADGET`, so plain `HSS` for the host side won't conflict. )

Reply:
HSS works here